auth-api-non-gms/src/main/java/com/omh/android/auth/nongms/domain/auth/AuthUseCase.kt

```diff
@@ -16,10 +16,10 @@
 
 package com.omh.android.auth.nongms.domain.auth
 
+import com.omh.android.auth.api.domain.utils.Pkce
```

Comment:
Hey @TranceLove. I'm working on ways for us to make this more efficient in terms of testing. If you have any ideas, please let me know.

Reply:
Hello @dzuluaga, very sorry for the late reply as I'm also sprinting on adding some OAuth2 providers based on this PR. Can't give much promise here though... but I hope I can give you some updates soon in a separate PR.

Regarding tests... sorry I don't have much idea here, but I think some integration test to mock the whole authentication flow would be a good idea. Will try to make some as I can see my implementations starts working :sweat_smile: